.gitlab-ci.yml

```diff
@@ -190,6 +190,21 @@ node-wrk:
   script:
     - *push_to_docker_hub
 
+gnupg:
+  <<:                              *docker_build
+  script:
+    - *push_to_docker_hub
+
+rpm:
+  <<:                              *docker_build
+  script:
+    - *push_to_docker_hub
+
+deb:
+  <<:                              *docker_build
+  script:
+    - *push_to_docker_hub
+
```

Comment:
Should we prefix those with `cleanroom` or `rel` for instance @TriplEight ?

Reply:
nah it looks fine, these images can be used out there, so generic names are better. I'd suggest prefixing them with the distro name/codename though.